common/components/org.wso2.carbon.data.agents.log4j/pom.xml

```diff
@@ -61,14 +75,22 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
-                        <!--<Private-Package>org.wso2.carbon.analytics.common.internal.*</Private-Package>-->
-                        <!--<Export-Package>-->
-                            <!--!org.wso2.carbon.analytics.common.internal.*,-->
-                            <!--org.wso2.carbon.analytics.common.spark.*-->
-                        <!--</Export-Package>-->
-                        <!--<Import-Package>-->
-                            <!--org.wso2.carbon.analytics.spark.core.*-->
-                        <!--</Import-Package>-->
+                        <Export-Package>
+                            org.wso2.carbon.data.agents.log4j.*
+                        </Export-Package>
+                        <Import-Package>
+                            org.apache.log4j.*,
+                            org.apache.log4j.spi,
+                            org.wso2.carbon.base.*,
+                            org.wso2.carbon.context.*,
+                            org.wso2.carbon.databridge.agent.*,
+                            org.wso2.carbon.databridge.commons.*,
+                            org.wso2.carbon.logging.service.*,
+                            org.wso2.carbon.user.*,
+                            org.wso2.carbon.utils.*,
```

Comment:
Although it is the best practice to put version range in Import, IMO, some of the package imports are better left without ranges. These include the core packages, e.g. log, carbon core packages, unless the component uses very specific function on the respective version. The reason we have to specify the version range is to avoid incorrect bundle wiring in OSGI container.